responsemanager/responsemanager.go

```diff
@@ -533,13 +343,47 @@ func (rm *ResponseManager) unpauseRequest(p peer.ID, requestID graphsync.Request
 	return nil
 }
 
+func (prm *processRequestMessage) handle(rm *ResponseManager) {
+	for _, request := range prm.requests {
+		key := responseKey{p: prm.p, requestID: request.ID()}
```

Comment:
General observation, ignore if no time to fix wholesale:
The contraction of `peerID` to `p` not only during varnaming but **also** in the struct-member names is fundamentally confusing. Especially given that the rest of the struct members have full-length names.

Reply:
that's a reasonable thought. I will address seperately.